build/scripts/build-impl.xml

```diff
@@ -344,11 +343,26 @@
         <echo message="Creating exist.jar ..."/>
         <jar basedir="${build.classes}" compress="true" jarfile="exist.jar">
             <manifest>
-                <attribute name="Project-Name" value="${project.name}"/>
-                <attribute name="Project-Version" value="${project.version}"/>
-                <attribute name="Project-Build" value="${DSTAMP}"/>
-                <attribute name="Project-CodeName" value="${project.codename}"/>
-                <attribute name="Git-commit" value="${git.commit}"/>
+                <attribute name="Implementation-Title" value="${project.name} Core"/>
+                <attribute name="Implementation-Version" value="${project.version}"/>
+                <attribute name="Implementation-Vendor" value="The eXist Project"/>
```

Comment:
I would say the original post's characterizations held up through that thread. To summarize: the community uses both (actually, more than two!) variants in different circumstances and according to individual preferences and conventions.